tests/dummy/app/templates/application.hbs

```diff
@@ -1,6 +1,6 @@
-{{ember-stripe-elements
-  blur="onBlur"
-  change="onChange"
-  focus="onFocus"
-  options=options
-}}
+{{#ember-stripe-element options=options as |stripeElement error|}}
+  {{#if error}}
+    <p>{{error.message}}</p>
+  {{/if}}
+  <button {{action "submit" stripeElement}}>Submit</button>
+{{/ember-stripe-element}}
```

Comment:
I still think we should support blockless usage as well, but as a first commit, either of the two is good for me.